src/main/java/com/sfuentes/ticket/Ticket.java

```diff
@@ -0,0 +1,35 @@
+package com.sfuentes.ticket;
+
+import com.sfuentes.grocery.Client;
+import com.sfuentes.grocery.Grocery;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+
+@Getter
+@RequiredArgsConstructor
+public class Ticket {
+
+  @NonNull
+  private final Grocery grocery;
```

Comment:
I need little bit of clarification here.
Here you have a property of type Grocery which I assume is from a previous exercise, so since the enclosing class is named Ticket it's a bit confusing to have a Grocery object inside a "Ticket", that implies the ticket contains a grocery.

Maybe it's fine and there is a valid reason to structure the code in this way, but probably we need to discuss it.